sbe-tool/src/main/java/uk/co/real_logic/sbe/generation/java/JavaGenerator.java

```diff
@@ -1276,10 +1284,11 @@ private void generateFixedFlyweightHeader(
         final String typeName,
         final String implementsString,
         final String buffer,
-        final String fqBuffer) throws IOException
+        final String fqBuffer,
+        final Set<String> additionalPackages) throws IOException
```

Comment:
`additionalPackages` -> `importedTypesPackages`?

Reply:
as you wish, though I'd like to point out, that those the not the only packes imported inside `generateFileHeader` so I figured it better to name it additional...

Reply:
no, only those that are identified by `scanPackagesToImport` and that do not match the current package name.

Reply:
renamed.